cluster/core/src/main/resources/reference.conf

```diff
@@ -30,3 +30,11 @@ akka.actor.serialization-bindings {
   "akka.actor.Address"        = akka-misc
   "akka.remote.UniqueAddress" = akka-misc
 }
+
+
+# Cluster distribution settings
+lagom.persistence.cluster.distribution {
```

Comment:
we should consider remove the `persistence` here and add a note on the migration guide. 

Cluster distribution has nothing to do with persistence. 

On the other hand, since this should better moved to akka core, we should try to not change config twice.